src/Apps/Artwork/Components/ArtistInfo.tsx

```diff
@@ -114,35 +146,59 @@ export class ArtistInfo extends Component<ArtistInfoProps> {
                   />
                 </>
               )}
+              {showArtistInsightsButton && (
+                <Flex flexDirection="column" alignItems="flex-start">
+                  <Button
+                    onClick={this.toggleArtistInsights}
+                    variant="secondaryGray"
+                    size="small"
+                    mt={1}
+                  >
+                    {buttonText}
+                  </Button>
+                </Flex>
+              )}
             </StackableBorderBox>
-            <MarketInsights
-              artist={this.props.artist}
-              border={false}
-              Container={Container}
-            />
-            <SelectedExhibitions
-              artistID={this.props.artist.id}
-              border={false}
-              totalExhibitions={this.props.artist.counts.partner_shows}
-              exhibitions={this.props.artist.exhibition_highlights}
-              ViewAllLink={
-                <a href={`${sd.APP_URL}/artist/${this.props.artist.id}/cv`}>
-                  View all
-                </a>
-              }
-              Container={Container}
-            />
+            {this.state.showArtistInsights && (
```

Comment:
One thing I noticed is that this version loses the bottom border in the case of multiple artists. (Possibly because of the extra fragment container here messing with the `StackableBorderBox`?).

See https://artsy-reaction.netlify.com/?path=/story/apps-artwork-page--artwork-with-multiple-artists (current state)
vs. https://deploy-preview-2098--artsy-reaction.netlify.com/?selectedKind=Apps%2FArtwork%20Page&selectedStory=Artwork%20with%20multiple%20artists&full=0&addons=0&stories=1&panelRight=0

![image](https://user-images.githubusercontent.com/2081340/54029695-479e9680-4177-11e9-94ea-71b86f83c129.png)

vs. 

![image](https://user-images.githubusercontent.com/2081340/54029701-4f5e3b00-4177-11e9-9bfe-5b650f62b00e.png)